extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -262,7 +262,7 @@ Schemas may be given a `name`, `description`, and `version`.
 
 ### Feature Tables
 
-A feature table stores property values in a parallel array format. Each property array corresponds to a class property. The values contained within a property array must match the data type of the class property. Furthermore, the set of property arrays must match one-to-one with the class properties. There is one exception - if a property is optional the feature table may omit that property.
+A feature table stores property values in a parallel array format. Each property array corresponds to a class property. The values contained within a property array must match the data type of the class property. Furthermore, the set of property arrays must match one-to-one with the class properties. There is one exception - if a property specifies a `noData` value, the feature table may omit that property.
```

Comment:
Maybe this belongs in the Cesium 3D Metadata specification, but it would be good to mention the rules for `noData`, including an implementation note about using an enum instead of a boolean if `noData` is needed.

Reply:
Currently this document does not describe the schema at all, and the referenced "Schemas" section of the Metadata Specification claims, "This document ... does not define a JSON schema." I think I understand the intention (JSON layout in EXT_feature_metadata and 3DTILES_metadata, semantic meaning in the core spec?) but it is a bit confusing to follow. Until we have a clear place for it, I'd rather not include a writeup of this specific property here just yet, but agree that such a writeup should be added in a language/clarity pass.